gridcapa-task-manager-app/src/main/java/com/farao_community/farao/gridcapa/task_manager/app/service/TaskDtoBuilderService.java

```diff
@@ -76,15 +82,15 @@ public List<TaskDto> getListTasksDto(LocalDate businessDate) {
         }
 
         tasks.stream()
-                .map(t -> createDtoFromEntityWithOrWithoutEvents(t, false))
+                .map(this::createDtoFromEntityNoLogs)
```

Comment:
I prefer to rename this method "createDtoFromEntityWithoutEvents" so  replace "NoLogs" by "WithoutEvents"